README.md

```diff
@@ -1,30 +1,99 @@
-# Configuration for Halide buildbots
+# Python environment
 
-To get started:
-
-Install Python 3.6+, python3-pip, and python3-venv, then run:
+To get started setting up _either_ a worker or a master, install Python 3.6+, python3-pip, and python3-venv, then create
+a virtual environment and install our Python dependencies:
 
 ```console
 $ python3 -m venv venv
 $ . venv/bin/activate
 $ pip install -r requirements.txt
 ```
 
-First, choose a buildbot password `$PASSWORD`. Then create and launch the build master:
+# Master configuration
+
+## Web server settings
+
+Using your production-quality web server of choice (Apache, Nginx, etc.), choose a URL at which to host the master. Call
+this `BUILDBOT_WWW`. Then, set up a reverse proxy for the buildbot webserver (on port 8012). For Apache, your
+configuration might look like:
+
+```
+ProxyPass /ws ws://localhost:8012/ws
```

Comment:
Note to self: we should add Nginx instructions here after we figure them out (since that's what our buildbot master is currently using)